packages/devtools/devtools/src/panels/client/DiagnosticsPanel/DiagnosticsPanel.tsx

```
@@ -74,7 +74,7 @@ export const DiagnosticsPanel = () => {
         info && (
           <div className='flex p-2 items-center text-sm font-mono gap-2'>
             {info.map((text, i) => (
-              <div key={i} className='w-flex items-center gap-1 cursor-pointer' onClick={() => handleCopy(text)}>
+              <div key={i} className='inline-flex items-center gap-1 cursor-pointer' onClick={() => handleCopy(text)}>
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)